### PR TITLE
initial dev commit

### PR DESCRIPTION
**Pull request by kgraaf**
Created this PR to discuss how the development of this library should proceed

### CODE REVIEW

File: akhali.py [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+    class _Month(_Control):
+        FULL = [ 'january', 'february', 'march', 'april', 'may',
+                 'june', 'july', 'august', 'september', 'october',
+                 'november', 'december' ]
+        ABBR = [ 'jan', 'feb', 'mar', 'apr', 'may', 'jun',
+                 'jul', 'aug', 'sep', 'oct', 'nov', 'dec' ]
+
+        def __init__(self, maproom, id, default, block, label):
+            self.default = default.lower()
```
Comment (kgraaf) [inline akhali.py]: Instead, only accept lower case, three letter abbreviated month name

File: akhali.py [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+
+    def _validate_id(self, id, kind):
+        if id not in self._ids_in_use.keys():
+            raise UndefinedId(id, kind)
+        elif self._ids_in_use[id] != kind:
+            raise WrongKind(id, self._ids_in_use[id], kind)
+        else:
+            pass
+
+    def _add_component(self, ctrl, block, label):
```
Comment (kgraaf) [inline akhali.py]: make name of "ctrl" consistent with component

File: akhali.py [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+
+class MaproomException(Exception):
+    pass
+
+
+
+class Maproom:
+    def __init__(self, title, prefix):
+        self.title = title
+        self.prefix = prefix
```
Comment (aaron-kaplan) [inline akhali.py]: Call this "urlpath"?

File: akhali.py [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+            'label': label,
+            'data': data,
+            'inputs': inputs,
+            'function': function,
+        })
+
+
+    # render/start
+    def render(self, server):
+        APP = dash.Dash(
```
Comment (aaron-kaplan) [inline akhali.py]: `APP` is now a local variable now, so it should be named `app`.

File: akhali.py [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+            __name__,
+            server=server,
+            url_base_pathname=f"/{self.prefix}/",
+            external_stylesheets=[
+                dbc.themes.BOOTSTRAP,
```
Comment (aaron-kaplan) [inline akhali.py]: interface, port, etc. should be configurable. See enacts app.py.

**Pull request merged**